3sidedcube/react-native-elements
# feat(switch): Add functionality and prop to determine touch area for switch row toggle

## How to test
1. For a switch row element, add the prop `toggleOnSwitchIconPressOnly`, when you press the switch element, it should only toggle when you press on the icon and not on the text.
2. Remove the prop, and expect the switch to toggle when you press anywhere on the switch row.

## Screen Recording

https://user-images.githubusercontent.com/37588380/173365161-4ba0fc88-959a-45f6-8e78-dc199a24cb85.mov
